lab/tree_builder.py

```diff
@@ -17,17 +19,45 @@ def __init__(self, root):
         self.root = root
         self.size = 0
```

Comment:
Can you actually change this to `1` since we are passing in a TreeNode during initialization. Apologize for the error.